src/index.ts

```diff
@@ -165,6 +166,7 @@ if (global.__thundraMasterModule__) {
         listeners,
         init,
         Foresight,
+        GenericWrapper,
```

Comment:
maybe it is better just export `wapper` method of `GenericWrapper` instead of exporting `GenericWrapper` completely.
If so, what about export `wapper` as `nodeWrapper`?